app/src/androidTest/java/com/h/pixeldroid/CameraTest.kt

```diff
@@ -0,0 +1,36 @@
+package com.h.pixeldroid
+
+import android.Manifest
+import androidx.fragment.app.testing.launchFragmentInContainer
+import org.junit.Rule
+import org.junit.Test
+import androidx.test.rule.GrantPermissionRule
+import com.h.pixeldroid.fragments.CameraFragment
+import kotlinx.android.synthetic.main.camera_ui_container.*
+import kotlinx.android.synthetic.main.fragment_camera.*
+
```

Comment:
Please avoid import *

Reply:
these are kotlin's imports from xml, I thought we agreed we made an exception for this